multiview/embed/gcca.py

```diff
@@ -59,14 +108,7 @@ def _center(self, X):
         centered_X -= mu
         return centered_X
 
-    def fit(
-        self,
-        Xs,
-        fraction_var=0.9,
-        sv_tolerance=None,
-        n_components=None,
-        tall=False,
-    ):
+    def fit(self, Xs):
         """
         Calculates a projection from each view to a latentent space such that
```

Comment:
typo

Reply:
See #45, also related to slack comments you made